hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), MSKTOP(i.s0.l()));
+}
```

Comment:
Looks like the other 'b' instructions use the .l() method.  See

    420 #define Y(vasm_opc, simd_w, vr_w, gpr_w, imm) \
    421 void Vgen::emit(const vasm_opc& i) {          \
    422   if (i.d.isSIMD()) {                         \
    423     emitSimdImmInt(a, i.s.simd_w(), i.d);     \
    424   } else {                                    \
    425     Vreg##vr_w d = i.d;                       \
    426     a->Mov(gpr_w(d), imm);                    \
    427   }                                           \
    428 }
    429 
    430 Y(ldimmb, ub, 8, W, MSKTOP(i.s.l())) // <<---
runtime/vm/jit/vasm-arm.cpp